yt_dlp/extractor/tencent.py

```diff
@@ -147,45 +178,53 @@ class VQQVideoIE(VQQBaseIE):
 
     _TESTS = [{
         'url': 'https://v.qq.com/x/page/q326831cny0.html',
-        'md5': '826ef93682df09e3deac4a6e6e8cdb6e',
+        'md5': '84568b3722e15e9cd023b5594558c4a7',
         'info_dict': {
             'id': 'q326831cny0',
             'ext': 'mp4',
             'title': '我是选手：雷霆裂阵，终极时刻',
             'description': 'md5:e7ed70be89244017dac2a835a10aeb1e',
             'thumbnail': r're:^https?://[^?#]+q326831cny0',
+            'format_id ': 'shd',
+            'has_drm ': False,
```

Comment:
VQQVideo tests are failing because you have a blank space at the end of the keys

Also multiple cases below

```python
'format_id '  -> 'format_id'
'has_drm '  -> 'has_drm'
```